packages/connector-chrome/tests/events.ts

```diff
@@ -9,7 +9,8 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { URL } from 'url';
 
-import { groupBy, every } from 'lodash';
+import groupBy = require('lodash/groupBy');
```

Comment:
Did you take a look at [`lodash-es`](https://www.npmjs.com/package/lodash-es)?

Reply:
Yes, but it won't work for us until we change our TypeScript build to emit ecmascript modules instead of commonjs (which requires all our dependencies to support ecmascript modules as well). 

Thus the cherry-picked approach is our only option at the moment.